src/mod/l4_thruster.h

```diff
@@ -12,20 +12,18 @@
 #include <chip.h>
 
 
-// Variables below will store value returned by READ SINGLE REGISTER request
+// Variables below will store value returned by read single register request
 // Output from READ SINGLE REGISTER can be a variable of either 1,2 or 4 bytes long.
 // Those global variables will represent LATEST RECEIVED VALUE returned by READ SINGLE REGISTER request
 extern uint8_t VALUE_UINT8;
 extern uint16_t VALUE_UINT16;
 extern uint32_t VALUE_UINT32;
 extern uint8_t READ_REQUEST_OK_FLAG;  // if checksum of parsed message was verified, then message is considered valid. Corresponding flag is set to 1
 extern uint8_t LATEST_ACCESSED_REGISTER; // Latest register to which SET or READ request have been made
-extern double ACTUAL_VALUE; // After extracting either uint8 or uint16 or uint32 value to obtain actual real value conversion multipliers have to be applied
+extern double ACTUAL_VALUE; // //ACTUAL VALUE represents real physical value returned by READ SINGLE REGISTER REQUEST
```

Comment:
Idea was to use is as "OBC registers" that would store latest parsed values from read request